wpilibj/src/main/java/edu/wpi/first/wpilibj/Resource.java

```diff
@@ -90,8 +90,8 @@ public int allocate(final int index) throws CheckedAllocationException {
 
   /**
    * Free an allocated resource. After a resource is no longer needed, for example a destructor is
-   * called for a channel assignment class, Free will release the resource value so it can be reused
-   * somewhere else in the program.
+   * called for a channel assignment class, Free will release the resource value, so it can be
```

Comment:
This reads better without the comma

Reply:
the comma should stay because it is two independent clauses but I will change it from 'Free' to 'this method'